examples/web-chat/src/App.tsx

```diff
@@ -9,7 +9,7 @@ import { useMessages, usePersistentNick } from "./hooks";
 
 const startTime = new Date();
 // Only retrieve a week of history
-startTime.setTime(Date.now() - 1000 * 60 * 60 * 24 * 7);
+startTime.setTime(Date.now() - 1000 * 60 * 30);
```

Comment:
Would be good to update the ccomment. @weboko

Reply:
I intend to roll it back and investigate.